packages/origin-ui-core/src/components/bundles/BundleItemEdit.tsx

```diff
@@ -0,0 +1,231 @@
+                                )
+                            }
+                        });
+                    }}
+                    initialValues={{
+                        volumeToBundle: EnergyFormatter.format(volumeToBundle)
+                    }}
+                >
+                    {(formikProps) => {
+                        const { isValid, setFieldValue } = formikProps;
+                        return (
+                            <Form translate="">
+                                <Grid item container style={{ marginTop: spacing(2) }}>
+                                    <Grid item style={{ flexGrow: 1 }}>
+                                        <Field name="volumeToBundle">
+                                            {({ field }) => (
+                                                <FormControl
+                                                    classes={{ root: classes.formControl }}
+                                                >
+                                                    <InputLabel
+                                                        classes={{ root: classes.formLabel }}
+                                                    >
+                                                        {t('bundle.info.editBundleVolume')}
+                                                    </InputLabel>
+                                                    <Input
+                                                        name="volumeToBundle"
+                                                        {...field}
+                                                        classes={{ input: classes.formInput }}
+                                                        type="string"
+                                                        endAdornment={
+                                                            <InputAdornment
+                                                                position="end"
+                                                                style={{ marginBottom: spacing(1) }}
+                                                            >
+                                                                <IconButton
+                                                                    onClick={() =>
+                                                                        setFieldValue(
+                                                                            'volumeToBundle',
+                                                                            0
+                                                                        )
+                                                                    }
+                                                                >
+                                                                    <HighlightOff
+                                                                        style={{ fill: '#878787' }}
```

Comment:
Do we need to have this color defined in the component?

Reply:
This is the color of the reset button. As I understand it shouldn't change
![image](https://user-images.githubusercontent.com/44746858/88515835-fa81c580-cff4-11ea-8f6d-255010ba3467.png)